roles/openshift_sanitize_inventory/tasks/main.yml

```diff
@@ -115,3 +115,14 @@
   when:
     - openshift_master_console_port is defined
     - openshift_master_console_port != (openshift_master_api_port | default(8443))
+
+- name: At least one master is schedulable
+  fail:
+    msg: >
+      No schedulable masters found, please remove 'openshift_schedulable=False' from all of your masters.
+  when:
+    - l_master_schedulable | length > 0
+    - true not in l_master_schedulable
```

Comment:
We should fail if False in this list, not 'not true'.

We want all masters to be schedulable, so they need to fail if any are not.

Reply:
Fixed